draft-robert-mimi-delivery-service.md

```diff
@@ -123,6 +127,9 @@ Client           Delivery Service      Queueing Service
 ~~~
 {: title="Client/Delivery Service communication with fanout" }
 
+In the federated case, messages to a "foreign" Delivery Service can be proxied
```

Comment:
We should re-use existing terminology and either call this "guest" or "participating" DS.